src/schema/v2/home/__tests__/home_page_fairs_module.test.js

```diff
@@ -1,40 +1,77 @@
-/* eslint-disable promise/always-return */
 import { runQuery } from "schema/v2/test/utils"
+import { nanoid } from "nanoid"
+import moment from "moment"
+import { range } from "lodash"
+
+// adding this dep explicitly so the tests run when we change the main file
+;() => {
+  require("../home_page_fairs_module")
+}
+
+const mockRunningFair = () => {
+  const id = nanoid()
```

Comment:
`Math.random().toString(26).slice(2)` 🤷

Reply:
Hm ok, I could use this to replace nanoid.